docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -7,46 +7,43 @@ markdown: kramdown
 ---
 
 # IBM FHIR Server - Search Configuration Overview
-The [FHIR Specification](https://www.hl7.org/fhir/r4/search.html) defines a set of searchable fields for each resource type, and corresponding RESTful API.  The IBM FHIR Server supports searching using the specification defined search parameters and using tenant-specific search parameters (extensions).
+The [FHIR Specification](https://www.hl7.org/fhir/r4/search.html) defines a set of searchable fields for each resource type. The IBM FHIR Server supports searching via both specification-defined search parameters and tenant-specific search parameters.
 
 Specifically, the IBM FHIR Server supports searching on additional fields, including:
-* fields that are defined in the base specification, which are not configured for search;  
-* extension elements that a tenant may add to a standard FHIR resource type; and  
-* attributes that you define as part of a custom resource type  
+* fields that are defined in the base specification, which are not configured for search; and
+* extension elements that a tenant may add to a standard FHIR resource type
 
-The IBM FHIR Server allows deployers to define search parameters on a tenant-specific basis. This allows each tenant to share an instance of the FHIR server while maintaining the ability to have their own set of search parameters.
+The IBM FHIR Server allows deployers to define search parameters on a tenant-specific basis. This allows each tenant to share an instance of the FHIR server while maintaining the ability to have their own set of search parameters. Additionally, specification-defined search parameters can be filtered out in order to avoid the cost of extracting and storing the corresponding indices.
 
-A tenant must provide a [Bundle](https://www.hl7.org/fhir/r4/bundle.html) of [SearchParameter](https://www.hl7.org/fhir/r4/searchparameter.html) resources that define the additional search parameters which describe the searchable field and define the FHIRPath expression for extraction.  For example, a tenant that extends the `Patient` resource type with the `favorite-color` extension, enables search on `favorite-color` by defining a SearchParameter as part of this bundle.
-
-Note, the [composite](https://www.hl7.org/fhir/r4/codesystem-search-param-type.html#search-param-type-composite) and [special](https://www.hl7.org/fhir/r4/codesystem-search-param-type.html#search-param-type-special) SearchParameter types are not supported.
+Tenant search parameters are defined via a [Bundle](https://www.hl7.org/fhir/r4/bundle.html) of [SearchParameter](https://www.hl7.org/fhir/r4/searchparameter.html) resources that define the additional search parameters which describe the searchable field and define the FHIRPath expression for extraction.  For example, a tenant that extends the `Patient` resource type with the `favorite-color` extension, enables search on `favorite-color` by defining a SearchParameter as part of this bundle.
```

Comment:
The markdown shows a backslash in the text here, not sure why:
...expression for extraction. \For example, a tenant that extends...

Reply:
I think I fixed this one previously